homeassistant/components/cloud/strings.json

```diff
@@ -13,5 +13,21 @@
       "logged_in": "Logged In",
       "subscription_expiration": "Subscription Expiration"
     }
+  },
+  "issues": {
+    "legacy_subscription": {
+      "title": "Legacy subscription detected",
+      "fix_flow": {
+        "step": {
+          "confirm": {
+            "title": "title inside",
```

Comment:
Let's just drop this title so it takes the main title.
```suggestion
```

Reply:
👍 This was a placeholder anyway